Trustroots/trustroots
# Generate fake test data

#### Proposed Changes

* generate data in tests automatically

#### Testing Instructions

* `git switch fake-test-data`
* `npm run test:client`
* see that the tests pass, especially the TribesPage component tests

There is an [ongoing discussion](https://github.com/trustroots/trustroots/pull/1169#discussion_r364297251) whether to write fixed test data or generate them. I'm not opinionated and I don't mind merging this or closing it unmerged. :)

Follow up of #1169
